Resources/Prototypes/CM14/Entities/Clothing/OuterClothing/USCMarmor.yml

```diff
@@ -0,0 +1,73 @@
+﻿- type: entity
+  parent: ClothingOuterArmorBasic
+  id: ClothingOuterArmorMarineM3
+  name: M3 Pattern Personal Armor
+  description: Standard Issue Armor used by USCM. Provides modest protection from bullet and melee attacks. It also has a shoulder-mounted directional flashlight for night time combat and a HUD to see their Marine roles. An all rounder in terms of protection with an additional 3 slots in the armor.
+  components:
+    - type: Sprite
+      sprite: CM14/Clothing/OuterClothing/Armor/M3armor.rsi
+      layers:
+        - state: icon
+        - state: light
+          visible: false
+    - type: Clothing
+      sprite: CM14/Clothing/OuterClothing/Armor/M3armor.rsi
+    - type: Armor
+      modifiers:
+        coefficients:
+          Blunt: 0.8
+          Slash: 0.8
+          Piercing: 0.4
+          Heat: 0.9
+    - type: ExplosionResistance
+      damageCoefficient: 0.80
+    - type: ToggleableLightVisuals
+      spriteLayer: light
+    - type: PointLight
+      enabled: false
+      radius: 3
+      energy: 2
+      mask: /Textures/Effects/LightMasks/cone.png
+      autoRot: true
+      netsync: false
+    - type: HandheldLight
+      addPrefix: true
+      blinkingBehaviourId: blinking
+      radiatingBehaviourId: radiating
+    - type: LightBehaviour
+      behaviours:
+        - !type:FadeBehaviour
+          id: radiating
+          interpolate: Linear
+          maxDuration: 2.0
+          startValue: 3.0
+          endValue: 2.0
+          isLooped: true
+          property: Radius
+          enabled: false
+          reverseWhenFinished: true
+        - !type:PulseBehaviour
+          id: blinking
+          interpolate: Nearest
+          maxDuration: 1.0
+          minValue: 0.1
+          maxValue: 2.0
+          isLooped: true
+          property: Radius
+          enabled: false
+    - type: Battery
+      maxCharge: 600 #lights drain 3/s but recharge of 2 makes this 1/s. Therefore 600 is 10 minutes of light.
+      startingCharge: 600
+    - type: BatterySelfRecharger
+      autoRecharge: true
+      autoRechargeRate: 2 #recharge of 2 makes total drain 1w / s so max charge is 1:1 with time. Time to fully charge should be 5 minutes. Having recharge gives light an extended flicker period which gives you some warning to return to light area.
+    - type: Storage
+      capacity: 10
```

Comment:
Upstream is phasing out this kind of capacity value, maybe comment it out and add the current upstream sizes in case they revert it but still allow this to be fixed easily anyway